ovara-ui/src/app/hooks/useFetchAlkamisvuodet.ts

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useState } from 'react';
+import { apiFetch } from '@/app/lib/ovara-backend/api';
+
+export function useFetchAlkamisvuodet() {
+  const [alkamisvuodet, setAlkamisvuodet] = useState<Array<string> | null>(
+    null,
+  );
+
+  useEffect(() => {
```

Comment:
Toimiihan se näinkin mutta tosiaan se react-query olisi vaihtoehto saada haettua api-data client-komponentteihin ilman näitä useEffectejä.